fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -2343,6 +2347,14 @@ public boolean isEnableLowCardinalityOptimize() {
         return enableLowCardinalityOptimize;
     }
 
+    public boolean isUseLowCardinalityOptimizeV2() {
+        return useLowCardinalityOptimizeV2;
```

Comment:
should check enableLowCardinalityOptimize = true?

Reply:
don't need, I checked in rule